File: materi/pertemuan_8/java/HeloReturn.java
public class HeloReturn {
    public static void main(String[] args) {
        System.out.println(hello());

        double c = tambah(2, 3);

        System.out.println(c);

    }

    static String hello() {
        return "Hello world";
    }

    static double tambah(int a, int b) {
        return a + b;
    }
}

File: materi/pertemuan_8/go/helo_return.go
package main
import ("fmt")

func main() {
	fmt.Println(hello())

	var c = tambah(2, 3)

	fmt.Println(c)
}

func hello() string {
	return "Hello world"
}

func tambah(a int, b int) int {
	return a + b
}